frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeAddTemplate.tsx

```diff
@@ -82,13 +107,20 @@ const CustomServingRuntimeAddTemplate: React.FC<CustomServingRuntimeAddTemplateP
             <ActionList>
               <ActionListItem>
                 <Button
-                  isDisabled={code === '' || loading}
+                  isDisabled={code === stringifiedTemplate || loading}
```

Comment:
We don't want to remove the `code === ''` check, if so, the button won't be disable is the box is empty.